### DISCUSSION TITLE
连封3个轻量级云服务器

### DISCUSSION THREAD

**Discussion by matrix3d** [v2fly/v2ray-core#2145] [Q&A]
在服务器ping baidu.com
ping不通
请问怎么配置能不被封ip

**Comment by Rangtian** [comment]
你是要从外面代理回国内？
你说被封ip，是各个测试站点都收不到包么？
https://ping.pe/
试试看国内丢不丢包？

**Comment by matrix3d** [comment]
我在国内，买的腾讯云的国外服务器。
使用过程中国，国外服务器远程 ping 国内 baidu.com 之类 的ping不通。
